ofl/notosanssymbols2/METADATA.pb

```diff
@@ -1,26 +1,23 @@
-name: "Noto Sans Symbols 2"
+name: "Noto Sans Symbols2"
 designer: "Google"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2020-11-19"
 fonts {
-  name: "Noto Sans Symbols 2"
+  name: "Noto Sans Symbols2"
   style: "normal"
   weight: 400
   filename: "NotoSansSymbols2-Regular.ttf"
   post_script_name: "NotoSansSymbols2-Regular"
   full_name: "Noto Sans Symbols2 Regular"
-  copyright: "Copyright 2017 Google Inc. All Rights Reserved."
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/symbols)"
 }
+subsets: "latin"
+subsets: "latin-ext"
 subsets: "menu"
 subsets: "symbols"
-is_noto: true
-sample_text {
-  masthead_full: "♠🖫🙥👍"
-  masthead_partial: "🏆🡽"
-  styles: "🌍✄✎ 🏔🏕🏌🏍🎭🎮 🯅🯆🯇🯉 🡢🡭🡱🡼 🯱🯲🯳🯴🯵🯶 🂮🂱🂲🂳"
-  tester: "✷✺✻✽ ⯮⮊⮫⬈⇨🡲 ♠♣🖪🖫 🨀🨁🨂🨃🨄🨅 🗪🗮🗰 ⯞⯟⯠⬥⬦⬧ ⢈⢉⢊⢋⣋⣌⣍ ◻◼🞔🞕🞖 𐇠𐇡𐇢𐇣𐇤𐇗𐇘"
-  poster_sm: "🡢🡱🏠💻🐿👁📽"
-  poster_md: "⌚✋⯧☔🛪🏟⛅🞽🕖🚲"
-  poster_lg: "🡽🨄"
```

Comment:
Is it okay to remove the sample text?

Reply:
From our last Quarterly onboarding on-call meeting, I would say not. IIUIC they said these metadata fields were used for symbols fonts for which we don't have a centralized sample text.

Reply:
Similar situation as in [Bamum](https://github.com/google/fonts/pull/5412#event-7589621703)